### ISSUE TITLE
[Feature Request] I'm wondering how to name these Paths

### ISSUE THREAD

**Issue by Katsuvera** [ffxivcode/AutoDuty#430]
### What problem are you trying to fix?

So this isn't really a feature request, but it's not a bug either. But I thought it better to discuss it here rather than on Discord so that it doesn't get lost.

I'll put a TL;DR at the end because this is going to involve some exposition.

So I reached ShB dungeons and quickly realized that they are a _very_ different beast than HW and StB dungeons. lol

First and foremost, enemies are scaled up and deal significant damage now. Stuff like avoidable AoEs and raidwides actually hurt.

In addition, somehow the NPCs have _scaled down_. For example, in Holminster Switch, Alphinaud's Physick heals for a measly ~5.5k HP (as a Tank, you have roughly 30k HP, so he heals you for 20% of your HP, at best). Compare this to the NPC healer in Doma Castle who easily heals for over 8k per Cure II (roughly 40% of your HP cuz Tanks have around 20k). NPC Tanks also feel squishier and seem to die significantly faster in double-pulls compared to their StB counterparts, which felt like they had some hidden passive that gave them more mit based on how many enemies were nearby. lol In one test run I did, Thancred died in a double-pull in Holminster in the time it took my WHM to cast one Esuna and one Holy. And then, in one of my early attempts at W2W pulling Dohn Mheg as a DPS, Urianger was simply not able to keep Thancred alive in one of the double-pulls, which resulted in a wipe. This was when I truly realized: Wow, these NPCs are nothing like their HW/StB counterparts.

Thirdly, enemies appear to have significantly reduced aggro-ranges and generally don't pull until you're right up in their face. This means it's more dangerous to double-pull as a DPS/Healer since you have to run all the way up to the second pack and likely tank a few autos while dragging it back to the first one / the NPC Tank.

TL;DR: W2W pulling as a DPS/Healer is inconsistent starting in ShB, since dungeons are scaled up and NPCs are scaled down, so they aren't worth doing.

As such, I've created the usual Tank W2W Paths with the default naming scheme of 「Tank W2W - タンクまとめ」 , and then I created Paths for DPS/Healers that just stop briefly in front of each pack in order to give the NPC Tank a moment to catch up and dash in first (much easier to do thanks to the reduced aggro range change). They do not W2W pull, and are not any faster than the base path, the only benefit they have is that the NPC Tank goes into pulls first which prevents Caster classes from having their first few casts interrupted due to being smacked by the enemies at the start of every pull lol (it also means one or two less heals that the NPC healer needs to throw on you).

TL;DR: I created Paths for DPS/Healers that stop before each pull and let the NPC Tank dash in first, but I don't really know what to name these Paths since they aren't W2W. Just 「Other - タンク以外」maybe? Technically they could just work _as_ the new default Path since they do almost the same thing, tho Tank classes would be pointlessly stopping for a second before each pack if they used this option.

_Alternatively_, because these DPS/Healer paths offer such a small / negligible benefit, it might not be worth implementing anything to differentiate them from the base path at all. Maybe I should just name them something like 「Katsu Alternative」and then put a note that just says it stop before pulls to let the Tank go in first, and ppl that care enough to read it can pick it if they want. 🤔 

Thanks for your time.

### What is the solution to the problem?

Deciding how to name DPS/Healer paths for ShB dungeons that aren't W2W.

**Comment by erdelf** [comment]
yeah we definitely need to come up with a more robust naming scheme.. also related to auto-selection.
Sounds like in your case it would be good for example to auto-select your new path on anything but tanks.

**Comment by nathnathn** [comment]
would role tags added to them be a idea?
also means autoduty would know what to use/not use automatically unless its overridden.
